test/coveralls-data.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  'created_at': '2016-06-14T16:08:59.479Z',
```

Comment:
put this data inside a property, something like:
{
  responseData: { created_at:...}
}